dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/dto/ApiHostListDto.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * DTO class for the official Cloudera API <a
+ * href="https://archive.cloudera.com/cm7/7.11.3.0/generic/jar/cm_api/apidocs/json_ApiHostList.html">json</a>
+ *
+ * <p>Note: The license for <a
+ * href="https://mvnrepository.com/artifact/com.cloudera.api.swagger/cloudera-manager-api-swagger/7.11.0">generated</a>
+ * code is unclear, the own model for public schema used instead of it.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ApiHostListDto {
+  private final List<ApiHostDto> hosts;
+
+  @JsonCreator
+  public ApiHostListDto(@JsonProperty(value = "hosts", required = true) List<ApiHostDto> hosts) {
+    this.hosts = hosts;
+  }
+
+  public List<ApiHostDto> getItems() {
```

Comment:
`getHosts`

Reply:
Well noted. In `json` the filed named `items`. Before current PR we didn't parse it and save as is for the later processing on server. But with this PR we start relay on the data from response, so we need to add a unit test to verify that we parse response properly. 
Similar to clusters tasks.

Reply:
Fixed.